### PR TITLE
Cql4bonnie unit test

### PR DESCRIPTION
**Pull request by edeyoung**
Testing title and title/description retriever for model.

### CODE REVIEW

File: test/unit/hqmf/cql/data_criteria_test.rb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    expected_title1 = "EncounterInpatient"
+    expected_title2 = "EncounterInpatient"
+    expected_description = "Encounter, Performed: EncounterInpatient"
+
+    assert title1 == expected_title1, string_match_error(title1, expected_title1)
+    assert title2 == expected_title2, string_match_error(title2, expected_title2)
+    assert description == expected_description, string_match_error(description, expected_description)
+  end
+
+  def string_match_error(str1, str2)
```
Comment (c-monkey) [inline test/unit/hqmf/cql/data_criteria_test.rb]: This approach for printing an error is actually redundant to the built-in `assert_equal` function's default behavior. For example:

```
assert_equal('Encounter InpatientXXX', title2)
```

with sample output:

```
Minitest::Assertion: Expected: "Encounter InpatientXXX"
  Actual: "Encounter Inpatient"
```

File: test/unit/hqmf/cql/data_criteria_test.rb [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    mat_5_2_filename = File.join(HQMF_CQL_ROOT, 'EHDI1a_v5_2_eMeasure.xml')
+    document = HQMF2CQL::Document.new(File.open(mat_5_2_filename).read)
+    data_criteria = document.all_data_criteria[0]
+
+    title1 = data_criteria.title
+    data_criteria.retrieve_title_and_description_for_model
+    title2 = data_criteria.title
+    description = data_criteria.description
+
+    expected_title1 = "Encounter Inpatient SNOMEDCT Value Set"
```
Comment (c-monkey) [inline test/unit/hqmf/cql/data_criteria_test.rb]: If you replace the `assert` with `assert_equal` as described below, then you can get rid of the 6 "expected_" variables, which will make the code cleaner. It made sense when you were using the `string_match_error` function

File: test/unit/hqmf/cql/data_criteria_test.rb [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    expected_title2 = "Encounter Inpatient"
+    expected_description = "Encounter, Performed: Encounter Inpatient"
+
+    assert title1 == expected_title1, string_match_error(title1, expected_title1)
+    assert title2 == expected_title2, string_match_error(title2, expected_title2)
+    assert description == expected_description, string_match_error(description, expected_description)
+
+    # Test MAT 5.3 export
```
Comment (c-monkey) [inline test/unit/hqmf/cql/data_criteria_test.rb]: I suggest refactoring the 2 blocks (Test MAT 5.2 export, and Test MAT 5.3 export) to share more code, they are almost exactly the same, except the filenames and the expected 3 strings

**Comment by edeyoung** [comment]
edited based on comments.

**Review by c-monkey — APPROVED** [review]
looks good, passes local execution

**Review by holmesie — APPROVED** [review]
Looks good!

**Pull request merged**